src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -48,6 +56,10 @@ public function __construct($name, NodeInterface $parent = null)
 
         $this->name = $name;
         $this->parent = $parent;
+
+        // Backwards compatibility for old property names, to be removed in 3.0
+        $this->normalizationClosures = &$this->normalizationCallbacks;
+        $this->finalValidationClosures = &$this->finalValidationCallbacks;
```

Comment:
Would it be an option to remove the `normalizationCallbacks` and `finalValidationCallbacks` properties and implementing `__get()` and `__set()` for them instead? This way we could also trigger deprecation notices when the deprecated properties are used.